src/style.scss

```diff
@@ -4,89 +4,129 @@
 // ---------------------------------------------------------------------
 
 // Import Auro tokens
-@import "./node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables";
-@import "./node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties";
-@import "./node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariableMap";
+@import './node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
```

Comment:
Why were these changed from `" "` to `' '`?   Something I go back and forth on as there are preferences on both sides. 

The generator creates a file using double-quotes. If we are to change direction in this, I'd prefer that we set that precedent from the generator, not in an individual PR.

Reply:
I think I see why you did this, there are some Sass files that are using single quotes. I still stand by the statement above. Let's pick a standard and make sure that everything is that way.

Reply:
Agreed, let's get into this with the followup on our conversation today to use a formatter consistently.